docs/headless-cms/03-03-markdown-documents.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Markdown documents
+type: Details
+---
+
+The DocsTopic or ClusterDocsTopic CR supports various documentation formats, including Markdown (`.md`) documents. Each `.md` file must consist of two parts: metadata and content. The following example illustrates the required structure: 
+
+```
+---
+metadata: {value or text string}
+another_metadata: {value or text string}
+---
+
+{The content of your document in **Markdown**.}
+```
+
+## Metadata
+
+Each Markdown document displayed in the Kyma Console requires metadata in a specific format called [Front Matter](https://forestry.io/docs/editing/front-matter/). 
+
+### Structure
+
+When you create a Markdown document, define its `title` and `type`. Place the metadata at the top of your `.md` file, and separate it with three dashes.
+
+```
+---
+title: {Document title}
+type: {Document type}
+---
+```
+
+- The `title` metadata defines the title of your document. 
+- The `type` metadata groups single documents together. Multiple documents that use the same `type` generate a grouping. For example, if you have multiple tutorials, you can group them below a navigation node called **Tutorials**.
+
+>**NOTE:** If there is only one document of a certain type, remove the `type` metadata, so that the document displays well in the UIs.
```

Comment:
```suggestion
>**NOTE:** If there is only one document of a certain type, remove the `type` metadata, so that the document displays well in the UI.
```